controllers/replica_set_controller.go

```diff
@@ -390,6 +404,14 @@ func (r *ReplicaSetReconciler) ensureService(mdb mdbv1.MongoDBCommunity) error {
 		r.log.Infof("The service already exists... moving forward: %s", err)
 		return nil
 	}
+	svc = buildService(mdb)
```

Comment:
parameterize `buildService` to be able to build the arbiters service without having to patch it before creation